### DISCUSSION TITLE
My experience with experimental-13.1-RELEASE

### DISCUSSION THREAD

**Discussion by Beda-Conky** [helloSystem/hello#370] [Show and tell]
![Screenshot_20220701_142236](https://user-images.githubusercontent.com/107292491/176955536-bd50d13f-dfdc-4a7a-90f4-01237d637286.png)

Hi, I am just curious about operating systems. I was using Linux distributions immersively for 10 years, and for recent personal problems I had to change, so. I gave the freebsd a chance I was waiting for the release 13.1 I installed with KDE5 on a separate disc, and used KDE5, almost everything work ok. And then when I installed to test the helloSystem 0.8.0 (Build 0H32) The result was impressive.

What works on my machine:

* Youtube works 100% only with Simple Browser (I don't know why)

What I haven't found the solution yet:

* I wish I could start my script for force the correct resolution of my screen as a automatic to when I turn on the PC.

Thank you so much for your commitment, and for building this beautiful operating system, now I am loving having a new experience with helloSystem.

Forgive me because of my words, English is not my main language, my main language is  Portuguese. Brazilian community is loving Hellosystem.

**Comment by probonopd** [comment]
Thanks for testing!

Normally the correct resolution should come up automatically. Please describe your screen: How large is it (diagonal inches), and which resolution (pixels x pixels) does it have natively?

**Comment by Beda-Conky** [comment]
18.5 inches, 1360x768@ 60Hz

https://www.lg.com/br/monitores/lg-W1943C-monitores-lcd-widescreen

every time I turn on the pc the resolution is fixed at 1024x768, so I need to force to the resolution indicated by the manufacturer

:)

**Reply by Beda-Conky** [reply]
I think natively it would be 1360x768, but no problem. thanks :)
